vendor/github.com/heimweh/go-pagerduty/pagerduty/event_orchestration_path.go

```diff
@@ -170,6 +176,7 @@ func (s *EventOrchestrationPathService) Update(id string, pathType string, orche
 func (s *EventOrchestrationPathService) UpdateContext(ctx context.Context, id string, pathType string, orchestrationPath *EventOrchestrationPath) (*EventOrchestrationPathPayload, *Response, error) {
 	u := orchestrationPathUrlBuilder(id, pathType)
 	v := new(EventOrchestrationPathPayload)
+	sanitizeOrchestrationPath(orchestrationPath)
```

Comment:
Please make sure to update vendored dependencies, since currently this file is an older version of your changes in https://github.com/heimweh/go-pagerduty/pull/139